__tests__/AzureSqlAction.test.ts

```diff
@@ -126,11 +134,13 @@ describe('AzureSqlAction tests', () => {
     });
 });
 
-function getInputs(actionType: ActionType) {
+function getInputs(actionType: ActionType, connectionString: string = '') {
```

Comment:
please add doc comment for this function

Reply:
Added